dataikuapi/dss/ml.py

```diff
@@ -643,6 +652,351 @@ def __repr__(self):
                                                           msg=self._internal_dict["message"])
 
 
+class DSSMLAssertionsParams(object):
+    """
+    Object that represents parameters for all assertions of a ml task
+    Do not create this object directly, use :meth:`DSSPredictionMLTaskSettings.get_assertions_params()` instead
+    """
+
+    def __init__(self, data):
+        self._internal_dict = data
+
+    def get_raw(self):
+        """
+        Gets the raw dictionary of the assertions parameters
+        :rtype: dict
+        """
+        return self._internal_dict
+
+    def get_assertion(self, assertion_name):
+        """
+        Gets a :class:`DSSMLAssertionParams` representing the parameters of the assertion with the
+        provided name (or None if no assertion has that name)
+        :param str assertion_name: Name of the assertion
+        :rtype: :class:`DSSMLAssertionParams` or None if no assertion has that name
+        """
+        for assertion_dict in self._internal_dict["assertions"]:
+            if assertion_dict["name"] == assertion_name:
+                return DSSMLAssertionParams(assertion_dict)
+        return None
+
+    def add_assertion(self, assertion_params):
+        """
+        Adds parameters of an assertion to the assertions parameters of the ml task.
+        :param object  assertion_params: A :class:`DSSMLAssertionParams` representing parameters of the assertion
```

Comment:
I don't understand when we should use `object` and when we should use the `DSS*class`. I mainly see usage of object after the `:param` keyword

Reply:
The docstring should be the closest to what we expect from the function, and is a poor replacement of static typing for python. It allows for example, when editing code in an IDE, to have a smarter autocomplete. `object` is used when we don't know the type of the corresponding parameter

Reply:
As discussed, because class names can become long, we will stick with the following notation in the doc
```
:param object object_name: A :class:`DSSMLClass_name` representing ...
```

<img width="714" alt="Screen Shot 2020-12-16 at 14 44 11" src="https://user-images.githubusercontent.com/18307978/102358117-6cac7400-3faf-11eb-8809-37eeffbde200.png">